console-extensions.json

```diff
@@ -11,7 +11,7 @@
     "type": "console.page/route",
     "properties": {
       "exact": true,
-      "path": ["/k8s/ns/:ns/add-broker", "/k8s/all-namespaces/add-broker"],
```

Comment:
Why are you removing route path for all-namespaces i.e. `"/k8s/all-namespaces/add-broker"` ? We need it.